.gitignore

```diff
@@ -1,23 +1,2 @@
-# Compiled class file
-*.class
-
-# Log file
-*.log
-
-# BlueJ files
-*.ctxt
-
-# Mobile Tools for Java (J2ME)
-.mtj.tmp/
-
-# Package Files #
-*.jar
-*.war
-*.nar
-*.ear
-*.zip
-*.tar.gz
-*.rar
-
-# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
-hs_err_pid*
```

Comment:
I guess that this was unintentional..

Reply:
Actually, since all the generated stuff lives in `out` it was intentional

Reply:
Even the crash logs are placed in `out` for Mill? That's neat if so!